plugins/kubernetes/test/models/kubernetes/deploy_executor_test.rb

```diff
@@ -20,6 +20,8 @@
   before do
     stage.update_column :kubernetes, true
     deploy.update_column :kubernetes, true
+
+    stub_request(:get, 'http://foobar.server/version').to_return(body: '{"gitVersion": "v1.5.0"}')
```

Comment:
this should go to the global stubs actually ... see kubernetes/test/test_helper.rb